6_vizprep/src/prep_plot_funs.R

```diff
@@ -13,3 +13,28 @@ write_put_fun <- function(plot_fun, ind_file){
   saveRDS(plot_fun, data_file)
   gd_put(ind_file, data_file)
 }
```

Comment:
If we go with the above suggestions, we can eliminate the need for `write_put_fun` entirely because we'll just be doing a `saveRDS(plot_fun, data_file)` without a `gd_put` or any need for an `ind_file` variable.